src/web/questions/Element.ts

```diff
@@ -1,14 +1,15 @@
-import { Actor, Question } from '@testla/screenplay';
+import { Actor } from '@testla/screenplay';
 import { Selector, SelectorOptions } from '../types';
 import { BrowseTheWeb } from '../abilities/BrowseTheWeb';
 import { CheckMode } from '../../types';
+import { BaseQuestion } from '../templates/BaseQuestion';
 
 type Mode = 'visible' | 'enabled' | 'text' | 'value' | 'count' | 'minCount' | 'checked';
 
 /**
  * Question Class. Get a specified state for a selector like visible or enabled.
  */
-export class Element extends Question<boolean> {
+export class Element extends BaseQuestion {
```

Comment:
Shall we also rename that to FrameEnabledQuestion?

Reply:
Done.

Reply:
Done.